### PR TITLE
Adding environment variable to control whether to show the hg branch …

### PR DESCRIPTION
**Pull request by Siecje**
…in the prompt

### CODE REVIEW

**Comment by astronouth7303** [comment]
Why turn off just HG branches? What about other VC's?

**Comment by Siecje** [comment]
I use Mercurial bookmarks not branches. I still want git branches and hg bookmarks.

Currently you could modify your ```$PROMPT``` and remove the ```{curr_branch: {}}```

**Comment by scopatz** [comment]
I am in favor of this, just needs a news item

**Comment by scopatz** [comment]
Thanks!

File: news/hg_branch.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+**Added:** The $VC_HG_SHOW_BRANCH environement variable to control whether to hide the hg branch in the prompt.
```
Comment (gforsyth) [inline news/hg_branch.rst]: This should be formatted as:

```console
**Added:** 

* The $VC_HG_SHOW_BRANCH environement variable to control whether to hide the hg branch in the prompt.
```

Comment (astronouth7303) [inline news/hg_branch.rst]:
```rst
``$VC_HG_SHOW_BRANCH``
```

Just to get some rst action going.

**Pull request merged**